packages/editor/src/store/selectors.js

```diff
@@ -1814,6 +1814,21 @@ export const getPostTypeLabel = createRegistrySelector(
 	}
 );
 
+/**
+ * Returns a post type support object on the current post
+ *
+ * @param {Object} state Global application state.
+ *
+ * @return {Object} The post type supports object.
+ */
+export const getPostTypeSupports = createRegistrySelector(
+	( select ) => ( state ) => {
+		const currentPostType = getCurrentPostType( state );
+		const postType = select( coreStore ).getPostType( currentPostType );
+		return postType?.supports ?? {};
+	}
+);
```

Comment:
Do we need to introduce a new public registry selector for the value the editor has been deriving forever? 😅

Reply:
I understand we should not add this selector, but what does "the editor has been deriving forever" mean? Doe we have this data selectable already?

Reply:
Components like `PostTypeSupportCheck` just duplicated the logic and derived value from the `supports` property.